pywinauto/unittests/test_common_controls.py

```diff
@@ -765,6 +765,7 @@ def test_expand_collapse_buttons(self):
 
         birds.click_input(where='button')
         self.assertEqual(birds.is_expanded(), True)
+        time.sleep(1.0)
```

Comment:
I think we can use `win32gui.GetDoubleClickTime()*1.5` or something like that.

Reply:
I tried 0.8 seconds, but in 1 of 30 builds the failed again with same error, so I decided to write 
`win32gui.GetDoubleClickTime() * 2.0` which corresponds to 1.0 second.